definitions/family-private/json/CaseEventToFields.json

```diff
@@ -1057,5 +1057,331 @@
     "PageDisplayOrder": 3,
     "PageFieldDisplayOrder": 28,
     "ShowSummaryChangeOption": "Yes"
+  },
+
+
+
+  {
+    "LiveFrom": "23/08/2021",
+    "CaseTypeID": "C100",
+    "CaseEventID": "solicitorCreate",
```

Comment:
Should it be part of the "people in the case" event or requirements are different now?

Reply:
You're right